snomed/com.b2international.snowowl.snomed.exporter.server/src/com/b2international/snowowl/snomed/exporter/server/net4j/SnomedExportServerIndication.java

```diff
@@ -561,7 +556,7 @@ private void executeCoreExport(final RevisionSearcher revisionSearcher, final OM
 			}
 			
 			logActivity("Exporting SNOMED CT descriptions into RF1 format");
-			new SnomedRf1DescriptionExporter(exportContext, revisionSearcher, includeExtendedDescriptionTypes).execute();
+			new SnomedRf1DescriptionExporter(exportContext, revisionSearcher, Concepts.REFSET_LANGUAGE_TYPE_UK, includeExtendedDescriptionTypes).execute();
```

Comment:
Could `LanguageSetting` be used in place of GB constants?

Reply:
Yes.